ffi/capi/examples/fixeddecimal/test.c

```diff
@@ -7,18 +7,18 @@
 #include <stdio.h>
 
 int main() {
-    ICU4XLocale* locale = ICU4XLocale_create("bn", 2);
-    ICU4XCreateDataProviderResult result = ICU4XDataProvider_create_static();
+    ICU4XLocale* locale = ICU4XLocale_create_bn();
+    ICU4XCreateFixedDecimalFormatDataProviderResult result = ICU4XFixedDecimalFormatDataProvider_create_static();
```

Comment:
thought: Given these changes should we split this example into an entirely separate one? That way we can have a "normal" FDF example as a real example, and then the size optimized one can be as weird as it likes